app/BUILD.bazel

```diff
@@ -374,6 +374,7 @@ Place your file here if:
 BINDING_ADAPTERS = [
     "src/main/java/org/oppia/android/app/databinding/ConstraintLayoutAdapters.java",
     "src/main/java/org/oppia/android/app/databinding/EditTextBindingAdapters.java",
+    "src/main/java/org/oppia/android/app/databinding/TextInputLayoutBindingAdapters.java",
```

Comment:
@BenHenning Added new file to BUILD.bazel file but the check is failing. 
Are there other steps which need to be done when we introduce new `BindingAdapters`

Reply:
@anandwana001 The error is because of `TextInputLayout`, i am not sure if you might need to mention it in this section
`# Library for all custom data-binding adapters under org/oppia/android/app/databinding.`

Reply:
Worked, Thanks